eng/scripts/Deploy-ServerJson.ps1

```diff
@@ -0,0 +1,80 @@
+#!/bin/env pwsh
```

Comment:
can you add a block comment explaining the purpose of the script for future maintainers